src/features/offerv2/components/MovieCalendar/MovieCalendar.tsx

```diff
@@ -39,16 +40,14 @@ type Props = {
 
 export const MovieCalendar: React.FC<Props> = ({ dates, selectedDate, onTabChange }) => {
   return (
-    <View>
-      <Spacer.TopScreen />
+    <Container>
       <BottomBar />
       <ScrollView horizontal showsHorizontalScrollIndicator={false}>
         {dates.map((date) => {
-          const day = extractDate(date)
-          const { weekday, fullWeekDay, dayDate, month, fullMonth, timestamp } = day
+          const { weekday, fullWeekDay, dayDate, month, fullMonth, timestamp } = extractDate(date)
           const isSelected = selectedDate.getTime() === timestamp
 
-          const { CalendarCell, CalendarText } = StatusPattern[isSelected ? 'selected' : 'default']
+          const { CalendarText } = StatusPattern[isSelected ? 'selected' : 'default']
```

Comment:
l'objet ne me semble plus utile

```suggestion
          const CalendarText = isSelected ? SelectedCalendarText : DefaultCalendarText
```